app/views/admin/settings/_feature_flags.html.erb

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Remove extra line 🙏

Reply:
Oh and don't forget to remove this extra line on a new commit, @aamarill

Reply:
I'll be sure to include this when I fix the tests that are currently failing.

Reply:
done!